DDG4/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ target_link_libraries(DDG4
   PUBLIC
   DD4hep::DDCore
   Geant4::Interface
+  Celeritas::corecel
```

Comment:
This must of course depend on DD4HEP_USE_CELERITAS, but actually make a DDG4Celeritas library.